composer.json

```diff
@@ -24,11 +24,12 @@
         "php": "^8.2.0"
     },
     "require-dev": {
-        "laravel-zero/framework": "^11.36.1",
         "illuminate/log": "^11.5",
         "illuminate/queue": "^v11.44.2",
+        "laravel-zero/framework": "^11.36.1",
         "mockery/mockery": "^1.6.12",
-        "pestphp/pest": "^3.7.1"
+        "pestphp/pest": "^3.7.1",
+        "tightenco/duster": "^3.2"
```

Comment:
I pulled out Duster because it makes the build huge. We can try to work with the Laravel Zero team to see if we can figure out how to exclude it from the build, but for now we can't have it here and exclude it from the build.

Reply:
Yeah. This was an artefact of me accidentally running the duster step during testing with then commits so the repo gets messed up. I tried to undo it but failed.